src/components/components/ProfilePageForm.jsx

```diff
@@ -0,0 +1,139 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import FormSection from "./components/FormSection";
+
+const formSections = [
+  {
+    title: "Account info",
+    inputFields: [
+        {label: "First Name",
+        name: "firstName",
+        required: true},
+        {label: "Last Name",
+        name: "lastName",
+        required: true},
+        {label: "User Name",
+        name: "userName",
+        required: true},
+        {label: "Email",
+        name: "email",
+        required: true}
+    ],
+  },
+  {
+    title: "Address",
+    inputFields:  [
+        {label: "Street",
+        name: "street",
+        required: false},
+        {label: "Street Number",
+        name: "streetNumber",
+        required: false},
+        {label: "City",
+        name: "city",
+        required: false},
+        {label: "Zip Code",
+        name: "zipCode",
+        required: false}
+    ],
+  },
+  {
+    title: "Contact info",
+     inputFields: [
+        {label: "Phone",
+        name: "phone",
+        required: true},
+        {label: "Website",
+        name: "website",
+        required: false},
+        {label: "User Name",
+        name: "userName",
+        required: false},
+        {label: "Email",
+        name: "email",
+        required: false}
+    ],
+  },
+  {
+    title: "Company info",
+    inputFields: [
+        {label: "Name",
+        name: "companyName",
+        required: false},
+        {label: "Catch Phrase",
+        name: "catchPhrase",
+        required: false}, 
+        {label: "Job Title", 
+          name: "jobTitle", 
+          required: false},
+        {label: "Business Statement",
+        name: "businessStatement",
+        required: false}
+    ],
+  },
+]
+
+const initialForm = {
+        firstName: "",
+        lastName: "",
+        email: "",
+        jobTitle: "",
+        street: "",
+        city: "",
+        id: "",
+        streetNumber: "",
+        userName: "",
+        zipCode: "",
+        phone: "",
+        website: "",
+        companyName: "",
+        catchPhrase: "",
+        businesStatement: ""
+}
+
+
+export default function ProfilePageForm( {setReloadContacts, reloadContacts} ) {
+  const [form, setForm] = useState(initialForm)
+
+  const contactId = useParams().id
```

Comment:
Found another odd behaviour: (sorry!)

When on `http://localhost:5173/profile/2` clicking either link on the page to get to profile 1 does take me to `http://localhost:5173/profile/1` but the info still remains as the person I just came from.

Reply:
No worries! That is strange - I'll go see what's going on